compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUEnums.td

```diff
@@ -128,6 +128,10 @@ def MFMA_F32_16x16x32_F8E4M3FNUZ  : I32EnumAttrCase<"MFMA_F32_16x16x32_F8E4M3FNU
 def MFMA_I32_16x16x32_I8  : I32EnumAttrCase<"MFMA_I32_16x16x32_I8", 0x0980>;
 def MFMA_I32_32x32x16_I8  : I32EnumAttrCase<"MFMA_I32_32x32x16_I8", 0x0981>;
 
+// "Virtual" non-native MFMA intrinsic where we modify the width of K-dim.
```

Comment:
Move this comment to the above comment and clarify that the `V` in `VMFMA` corresponds to Virtual. We don't need to pick out the K-dim here, instead just explain that enums with `V` are emulating a larger MMA with smaller ones.

Reply:
done, I think